content/applications/hr/payroll/work_entries.rst

```diff
@@ -80,95 +91,245 @@ there are any errors on the work entry, such as required fields being left blank
 required to be resolved before payslips can be generated.
 
 Any work entry that has a conflict to be resolved is indicated on the main :guilabel:`Work Entry`
-overview dashboard, located by going to :menuselection:`Payroll --> Work Entries --> Conflicts`,
-where only conflicts needing resolution are shown.
+dashboard, which can be accessed by navigating to :menuselection:`Payroll app --> Work Entries -->
+Work Entries`. Only conflicts needing resolution are shown by default.
 
-.. image:: work_entries/conflicts.png
+Conflicts are indicated with an orange triangle in the top-left corner of each individual work
+entry. Click on an individual work entry to see the date and time for the specific work entry, then
+click :guilabel:`Edit` to view the conflict details in a pop-up window.
+
+.. image:: work_entries/conflict-pop-up.png
    :align: center
-   :alt: Conflicts dashboard view showing all employee's conflicts in work entries.
+   :alt: A row of conflicts, with one entry showing details for the conflict.
+
+The conflict is briefly explained in an orange text box in the :guilabel:`Open` pop-up window that
+appears.
 
-Conflicts are indicated with an orange triangle in the top left corner of each individual entry.
-Click on an individual work entry to see the conflict details in a pop-up.
+The :guilabel:`Description`, :guilabel:`Employee`, and :guilabel:`Work Entry Type` are listed on
+the left side of the pop-up window. The :guilabel:`From` and :guilabel:`To` date and time range, as
+well as the total time (in hours), appears in the :guilabel:`Duration` field on the right side.
 
-The conflict is briefly explained in an orange text box.
+If the conflict is due to a time off request that has not been approved yet, a :guilabel:`Time Off`
+field appears on the left side, with the type of time off requested in the description.
 
-.. image:: work_entries/conflict-detail.png
+.. image:: work_entries/conflict-details.png
    :align: center
-   :alt: Details for a conflict appear in the pop-up.
+   :alt: The detailed conflict pop-up window that appears when Edit is clicked.
+
+Time off conflicts
+------------------
+
+The most common work entry conflicts are for time off requests that have been submitted, but not yet
+approved, which results in duplicate work entries for that employee (one for time off and another
+for regular work).
+
+If there is a conflict because a time off request is in the system for the same time that a regular
+work entry already exists, the time off request is entered in the :guilabel:`Time Off` field.
+
+Resolve on work entry
+~~~~~~~~~~~~~~~~~~~~~
+
+The time off conflict can be resolved either on the work entry pop-up window, or on a detailed time
+off request pop-up window.
```

Comment:
I would move this portion *just* above the 'Resolve on work entry' section, as a sort of lead-in to the 2 sections that follow.